### PR TITLE
i18n(zh-cn): update environment-variables.mdx and configuring-astro.mdx

### PR DESCRIPTION
**Pull request by magnum-zx**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description
update environment-variables.mdx and configuring-astro.mdx

### CODE REVIEW

**Comment by liruifengv** [comment]
I have a draft PR for `configuring-astro.mdx`.https://github.com/withastro/docs/pull/3010/files

File: src/content/docs/zh-cn/guides/environment-variables.mdx [hunk 1]
```diff
@@ -23,32 +28,51 @@ Astro 包括了几个开箱即用的环境变量:
...
 ```
-
-```ini
-.env                # 加载所有变量
-.env.local          # 加载所有变量，被 git 忽略
-.env.[mode]         # 只在特定模式下加载
-.env.[mode].local   # 只在特定模式下加载，被 git 忽略
-```
-
+更多关于`.env` 文件的信息，[请参阅 Vite 文档](https://vitejs.dev/guide/env-and-mode.html#env-files)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/environment-variables.mdx:52]: `Vite` doc can use `https://cn,vitejs.dev`

File: src/content/docs/zh-cn/guides/environment-variables.mdx [hunk 2]
```diff
@@ -14,7 +15,11 @@ SECRET_PASSWORD=password123
 PUBLIC_ANYBODY=there
 ```
 
-在这个示例中，`PUBLIC_ANYBODY`（通过 `import.meta.env.PUBLIC_ANYBODY` 访问) ，而 `SECRET_PASSWORD`（通过 `import.meta.env.SECRET_PASSWORD` 访问）将只对服务端可见。
+在这个示例中，`PUBLIC_ANYBODY`（通过 `import.meta.env.PUBLIC_ANYBODY` 访问） ，而 `SECRET_PASSWORD`（通过 `import.meta.env.SECRET_PASSWORD` 访问）将只对服务端可见。
+
+:::caution
+`.env` 文件不会加载到[配置文件](/en/guides/configuring-astro/#environment-variables)中。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/environment-variables.mdx:21]:
```suggestion
`.env` 文件不会加载到[配置文件](/zh-cn/guides/configuring-astro/#环境变量)中。
```

File: src/content/docs/zh-cn/guides/environment-variables.mdx (hunk 1 above)
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/environment-variables.mdx:52]: `Vite` doc can use `https://cn,vitejs.dev`
```suggestion
更多关于`.env` 文件的信息，[请参阅 Vite 文档](https://cn.vitejs.dev/guide/env-and-mode.html#env-files)。
```

**Comment by liruifengv** [comment]
Somewhere need update. Such as `查看官方的环境变量示例了解最佳实践。`

**Review by liruifengv — APPROVED** [review]
LGTM, thanks for your work! @magnum-zx

**Review by Yan-Thomas — APPROVED** [review]
Nice work, thanks @magnum-zx! Also thanks @liruifengv for the review 🙌

**Pull request merged**